test/testBlocks.jl

```diff
@@ -705,6 +693,7 @@ function testGroup!(fg, v1, v2, f0, f1)
         @test lsfPriors(fg) == [:af1]
 
         #TODO add test, don't know what we want from this
+        # desire is to list all factor types present a graph.
```

Comment:
I'll need more than that, it currently returns a strange dictionary, can it not just by a `Vector{Symbol}`